.github/workflows/build.yml

```diff
@@ -11,6 +11,66 @@ on:
 # We build for other platforms except linux which is already covered in build-and-test.
 # Also, We're not running tests here as those are already covered with linux build.
 jobs:
+  Get-CI-Image-Tag:
+    runs-on: ubuntu-latest
+    outputs:
+      ci-image-version-linux: ${{ steps.step-ci-image-version-linux.outputs.ci-image-version-linux }}
+    steps:
+      - name: Install crane
+        uses: iarekylew00t/crane-installer@v1
+        with:
+          crane-release: v0.15.2
+      - name: Checkout opensearch-build repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'opensearch-project/opensearch-build'
+          ref: 'main'
+          path: 'opensearch-build'
+      - name: Get ci image version from opensearch-build repository scripts
+        id: step-ci-image-version-linux
+        run: |
+          crane version
+          CI_IMAGE_VERSION=`opensearch-build/docker/ci/get-ci-images.sh -p centos7 -u opensearch -t build | head -1`
+          echo $CI_IMAGE_VERSION
+          echo "ci-image-version-linux=$CI_IMAGE_VERSION" >> $GITHUB_OUTPUT
+
+  Build-CCR-Linux:
+    strategy:
+      matrix:
+        java: [11, 17]
```

Comment:
Are we building for both java 11 & 17 now? earlier we were doing only 17.

Reply:
We are running tests on both 11 and 17 so we can build on both